lnrpc/routerrpc/router.proto

```diff
@@ -22,6 +22,12 @@ service Router {
     */
     rpc TrackPaymentV2 (TrackPaymentRequest) returns (stream lnrpc.Payment);
 
+    /*
+    SubscribePayments returns an update stream for every payment.
+    */
+    rpc SubscribePayments (SubscribePaymentsRequest)
```

Comment:
For consistency (within this sub-server...), should it be `TrackPayments` or `TrackAllPayments`?

Reply:
Yes, probably right, updated the name